scripts/network.py

```diff
@@ -0,0 +1,20 @@
+from ape import networks
+import ape
+
+#Default network is starknet:local:starknet
+print(ape.networks.provider.network.ecosystem.name)
+
+with networks.parse_network_choice("ethereum:local:test"):
```

Comment:
Consider putting this all in a `main()` method.. It has better support, I think you may get a warning otherwise.

```python
def main():
   everything
```